go/vt/sqlparser/sql.y

```diff
@@ -2777,6 +2778,20 @@ length_opt:
     $$ = NewIntVal($2)
   }
 
+var_length_opt:
```

Comment:
How do you feel about naming this `char_length_opt`? Looking at where it's used, that seems like a good, self-descriptive name and it matches up nicely with the other `length_opt` variants (e.g. `float_length_opt` and `decimal_length_opt`).

Reply:
I agree with you, it will be better if the name is char_length_opt for keeping the consistency